### DISCUSSION TITLE
Plugin KT for tabby

### DISCUSSION THREAD

**Discussion by rahulsinghk** [Eugeny/tabby#7672] [General]
hi Tabby team. 
there are a few plugins I would like to try to make myself, 
but I don't know how to get started, and there is not much info about it online :(

Please shed some light on the basic process of constructing the plugin. 

I appreciate any help you can provide.

**Comment by Eugeny** [comment]
Hey, have you checked out https://github.com/Eugeny/tabby/blob/master/HACKING.md ?
